### ISSUE TITLE
0.9.0 - ipcluster not being cleaned up

### ISSUE THREAD

**Issue by isaacovercast** [dereneaton/ipyrad#333]
After running step 1 twice and then `ps -ef | grep ipcluster` I see this:

```
casey    23446     1  5 23:29 ?        00:00:07 /home/casey/miniconda2/envs/py36/bin/python /home/casey/miniconda2/envs/py36/bin/ipcluster start --daemonize --cluster-id=ipyrad-cli-397 --engines=Local --profile=default --n=40
casey    23979     1  5 23:30 ?        00:00:06 /home/casey/miniconda2/envs/py36/bin/python /home/casey/miniconda2/envs/py36/bin/ipcluster start --daemonize --cluster-id=ipyrad-cli-758 --engines=Local --profile=default --n=40
```

ipcluster isn't being shut down after completion of step 1.

**Comment by isaacovercast** [comment]
As a side note, I often found it very useful to know which ipcluster instance was associated with which ipyrad process when the `cluster-id` flag used the PID of ipyrad as the identifier. Using random numbers is fine, but the PID I think was more reliable and useful... Just a thought.

**Comment by eaton-lab** [comment]
I think I changed it so that if an ipcluster instance that was started with 'auto=True' when using the API did fail to get shut down then it wouldn't raise an error if we tried to start a new one again with the same cluster-id. It was a sloppy solution though, since ideally we don't want to leave any orphaned clusters running.

I thought they were getting cleaned up now reliably, but I haven't been testing the CLI.

I say we change it back to pids.
